flink-dist/src/main/flink-bin/bin/jobmanager.sh

```diff
@@ -40,6 +40,10 @@ if [[ $STARTSTOP == "start" ]] || [[ $STARTSTOP == "start-foreground" ]]; then
     # Add JobManager-specific JVM options
     export FLINK_ENV_JAVA_OPTS="${FLINK_ENV_JAVA_OPTS} ${FLINK_ENV_JAVA_OPTS_JM}"
     parseJmJvmArgsAndExportLogs "${ARGS[@]}"
+    if [[ $? -ne 0 ]]; then
+      echo "[ERROR] Could not get JVM parameters and dynamic configurations properly."
+      exit 1
+    fi
```

Comment:
Isn't this a bit redundant given that we do the check in `parseJmJvmArgsAndExportLogs`?